CHANGELOG.md

```diff
@@ -3,6 +3,13 @@
 The AWS AppSync SDK for iOS enables you to access your AWS AppSync backend and perform operations like `Queries`, `Mutations` and `Subscriptions`. The SDK
 also includes support for offline operations.
 
+## 2.14.0
+
+### Bug Fixes
+
+* Fix a bug where delta sync was not correctly storing/ retrieving the `lastSyncTime`. See [issue #232](https://github.com/awslabs/aws-mobile-appsync-sdk-ios/issues/232)
+* **Breaking API Change** To fix the delta sync logic, there was a change in the hashing function used internally. This change can cause the existing app to ignore the cache for the first sync and fetch data using base query. Subsequent sync operation should work as normal.
```

Comment:
What happens to any existing cache data? Will it be re-used, overwritten, ignored and left to grow stale?

Reply:
Existing cache data will be ignored.